### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by hershpa**
Optimize and clean up the readme content for clarity. 
Signed-off-by: Hersh Pathak hersh.pathak@intel.com

### CODE REVIEW

**Comment by hershpa** [comment]
Hi @chaitanya1731 and @uMartinXu, can you please help review this PR?

**Comment by chaitanya1731** [comment]
PR looks good to me.

File: README.md [hunk 1]
```diff
@@ -5,26 +5,26 @@ This project is currently under active development, and as such, all source code
...
+- **Driver Upgrade Scenario** - There is a new release from Intel GPU Drivers. In this scenario, Intel data center GPU driver Dockerfile needs to be updated to include the new release, Intel data center driver container image will be built and packaged using the Dockerfile. After testing and certifying steps, the new image should be released on Red Hat Ecosystem Catalog.    
+- **Kernel Upgrade Scenario** - [Red Hat Enterprise Linux CoreOS (RHCOS)](https://docs.openshift.com/container-platform/4.12/architecture/architecture-rhcos.html) kernel is upgraded in a new OCP release. In this scenario, the new data center GPU driver container image should be built on the new kernel version and its corresponding driver toolkit base image. This mapping is automatically taken care of in the Dockerfile via the [DTK_AUTO build argument](https://github.com/rh-ecosystem-edge/kernel-module-management/blob/main/docs/enhancements/using-dtk-for-building-driver-containers.md#dockerfile-examples) which dynamically retrieves the appropriate driver toolkit base image. The new image also needs to be tested, certified and then released on Red Hat Ecosystem Catalog.
```
Comment (uMartinXu) [inline README.md]: driver toolkit -> Driver Toolkit

Comment (hershpa) [reply README.md]: Done

**Comment by hershpa** [comment]
This PR looks ready if no other concerns.

**Comment by uMartinXu** [comment]
The patch looks good to me.

**Pull request merged**